osutil/io_test.go

```diff
@@ -246,41 +246,58 @@ func (ts *AtomicSymlinkTestSuite) TestAtomicSymlink(c *C) {
 		c.Check(exp, Equals, target)
 	}
 
+	checkLeftoverFiles := func(sym string, exp []string) {
```

Comment:
This could be just a function on the suite or standalone. Not a strong opinion though.

Reply:
Yeah I thought about that, but I decided to just follow @bboozzoo's original implementation with the `mustReadSymlink` function.